expression/builtin_cast_vec.go

```diff
@@ -577,11 +577,49 @@ func (b *builtinCastTimeAsTimeSig) vecEvalTime(input *chunk.Chunk, result *chunk
 }
 
 func (b *builtinCastTimeAsStringSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinCastTimeAsStringSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	var buf *chunk.Column
+	var err error
+	if b.args[0].GetType().Tp == mysql.TypeTimestamp {
+		buf, err = b.bufAllocator.get(types.ETTimestamp, n)
+	} else if b.args[0].GetType().Tp == mysql.TypeDatetime {
+		buf, err = b.bufAllocator.get(types.ETDatetime, n)
+	}
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalTime(b.ctx, input, buf); err != nil {
+		return err
+	}
+
+	sc := b.ctx.GetSessionVars().StmtCtx
+	vas := buf.Times()
+	result.ReserveString(n)
+	for i, v := range vas {
```

Comment:
`v` is never used. We can use `for i:=0; i < n; i++`

Reply:
v is used. 
res, err = types.ProduceStrWithSpecifiedTp(v.String(), b.tp, sc, false)